.github/workflows/e2e-sample-apps-tests.yml

```diff
@@ -217,6 +220,9 @@ jobs:
       CYPRESS_CI: true
       TEST_SUITE: ${{ matrix.test_suite }}
       SAMPLE_APP_BRANCH_NAME: ${{ needs.get-sample-apps-data.outputs.branch_name }}
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
I'm under the impression that [the default permission](https://github.com/metabase/metabase/actions/runs/14731317441/job/41346049431#step:1:20) which is `write` should be enough, unless we want to restrict the actions for some reason.

My theory is that we don't need this line.

```suggestion
```

Reply:
I did the same as was done by @oreonl in his PR. But let's try changing it.